neurodamus/node.py

```diff
@@ -440,13 +440,13 @@ def compute_load_balance(self):
         circuit = self._base_circuit
         for name, circuit in self._extra_circuits.items():
             if circuit.get("PopulationType") != "virtual":
+                logging.info("Activating experimental LB for Sonata circuit '%s'", name)
```

Comment:
```suggestion
                logging.info(f"Activating experimental LB for Sonata circuit {name}")
```

I find this easier to read. No problem in any case, this is just personal preference

Reply:
that would case a lint; logging, for efficiency reasons, suggests you pass each of the params; that way the string isn't interpolated unless the logging level is actually using it.